modules/kargoBidAdapter.js

```diff
@@ -274,19 +274,16 @@ function getUserSyncs(syncOptions, _, gdprConsent, usPrivacy, gppConsent) {
     return syncs;
   }
   if (syncOptions.iframeEnabled && seed && clientId) {
-    for (let i = 0; i < CERBERUS.SYNC_COUNT; i++) {
-      syncs.push({
-        type: 'iframe',
-        url: CERBERUS.SYNC_URL.replace('{UUID}', clientId)
-          .replace('{SEED}', seed)
-          .replace('{INDEX}', i)
-          .replace('{GDPR}', gdpr)
-          .replace('{GDPR_CONSENT}', gdprConsentString)
-          .replace('{US_PRIVACY}', usPrivacy || '')
-          .replace('{GPP_STRING}', gppString)
-          .replace('{GPP_SID}', gppApplicableSections)
-      });
-    }
+    syncs.push({
```

Comment:
`syncs` still needs to be an array
https://docs.prebid.org/dev-docs/bidder-adaptor.html#registering-user-syncs